class/practice2.1/AdvancedHelloWorld.cpp

```diff
@@ -1,110 +1,94 @@
 #include <string>
-#include <vector>
 #include <iostream>
 #include <fstream>
 #include <sstream>
+#include <map>
 
-std::vector<std::string> splitString(const std::string& str, char delimiter) {
-    std::vector<std::string> tokens;
-    std::stringstream ss(str);
-    std::string token;
-
-    while (std::getline(ss, token, delimiter)) {
-        tokens.push_back(token);
-    }
-    return tokens;
+namespace {
+    const std::string bread = "bread";
+    const std::string deleteCommand = "delete";
 }
-int main(int argc, char* argv[])
-{
-    std::string argument2;
-        
-    std::string WordToFind;
-    if (argc < 2) {
-        std::cout << "Please provide a word to find." << std::endl;
-        return 0;
-    }
-    WordToFind= argv[1];
-    if (argc>2)
-    {
-        argument2 = argv[2];
+class Database {
+private:
+    std::map<std::string, int> statsData;
```

Comment:
Minor: usually it is better to move private part below and start class with public part.
Also it can be a good idea to create Database.hpp and Database.cpp to split declaration and implementation and so on.